packages/phpstan-extensions/src/Testing/AbstractServiceAwareRuleTestCase.php

```diff
@@ -23,7 +23,19 @@ public function analyse(array $filePaths, array $expectedErrorsWithLines): void
     {
         try {
             parent::analyse($filePaths, $expectedErrorsWithLines);
-        } catch (Throwable $throwable) {
+        } catch (ExpectationFailedException $throwable) {
```

Comment:
@samsonasik It is better to aim for specific exceptoin, as catching `Throwable` makes every test pass even if they fail. The narrower, the better. That's why I check fro the exception message content.

Reply:
@TomasVotruba thank you 👍